Signal/src/Models/TSMessageAdapaters/TSGenericAttachmentAdapter.m

```diff
@@ -100,7 +95,7 @@ - (UIView *)mediaView
 {
     if (_cachedMediaView == nil) {
         CGSize viewSize = [self mediaViewDisplaySize];
-        UIColor *textColor = (self.incoming ? [UIColor blackColor] : [UIColor whiteColor]);
+        UIColor *textColor = (self.incoming ? [UIColor colorWithWhite:0.2 alpha:1.f] : [UIColor whiteColor]);
```

Comment:
Shoot I deleted your comment while trying to delete mine.  Here's my response.

It's a great question. There's a lot of "ink" within the incoming attachment bubbles and I felt like it was "too much" and "not hierarchical enough" when all of the ink was black.  I thought we should ever-so-slightly de-emphasize the filename and even further de-emphasize the file size labels in order to say "the most important thing here is the play/pause or attachment icon which indicates that this is an attachment" and "also, here is the filename", "lastly, the file size is the least important detail".  I think this is sort of what designers call a hierarchy.

Notice that I treated incoming and outgoing attachments differently.  IMO White-on-blue doesn't "overwhelm with drabness" in the say way that black-on-gray does, so I used full-on white for outgoing filenames and slightly de-emphasized the subtitle.

Obviously this is all subjective.  WDYT?

### Before 

![attachment colors before](https://cloud.githubusercontent.com/assets/625803/25664018/5277318c-2fe7-11e7-96eb-b4888841b39c.png)

### After

![attachment colors after](https://cloud.githubusercontent.com/assets/625803/25664015/4ffecbb8-2fe7-11e7-8c8b-d8a15d64252c.png)